### ISSUE TITLE
sql-server-2014-alwayson-existing-vnet-and-ad stop during cluster configuration

### ISSUE THREAD

**Issue by BluedeepGit** [Azure/azure-quickstart-templates#5238]
sql-server-2014-alwayson-existing-vnet-and-ad https://github.com/Azure/azure-quickstart-templates/tree/master/sql-server-2014-alwayson-existing-vnet-and-ad

### Issue Details
The deploy works well until the cluster configuration phase with the following error:

VM has reported a failure when processing extension 'CreateCluster'. Error message: "DSC Configuration 'CreateFailoverCluster' completed with error(s). Following are the first few: Check the spelling of the cluster name. Otherwise, there might be a problem with your network. Make sure the cluster nodes are turned on and connected to the network or contact your network administrator. PowerShell DSC resource MicrosoftAzure_xCluster  failed to execute Set-TargetResource functionality with error message: The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: The computer 'xxxxx.xxxx.xxxx' is joined to a cluster. 
 Check the spelling of the cluster name. Otherwise, there might be a problem with your network. Make sure the cluster nodes are turned on and connected to the network or contact your network administrator.".

One sql server node is completely configured … the other one seem have the default configuration with the master and tempdb still in the default folders.

what's went wrong? How can solve the problem?

Thanks

**Comment by kjk129** [comment]
Is your Domain Controller VM running?

**Comment by BluedeepGit** [comment]
Yes domain controller is up and running….the cluster start configuration register name on domain and dns, add nodes, and configure witness share, but after then the template stop with the error above...
